api/admin/controller/storage_services.py

```diff
@@ -5,7 +5,8 @@
 from flask_babel import lazy_gettext as _
 from api.admin.problem_details import *
 from core.model import (
-    ExternalIntegration
+    ExternalIntegration,
+    get_one
```

Comment:
I couldn't update the name of a Storage integration because it was missing this import.